storm/src/main/storm/mesos/MesosNimbus.java

```diff
@@ -338,6 +345,27 @@ private MesosSchedulerDriver createMesosDriver() throws IOException {
     return driver;
   }
 
+  private boolean isAccepted(Offer offer) {
+    return isHostAccepted(offer.getHostname())
+            && isAttributeAccepted(offer.getAttributesList());
+  }
+
+  private boolean isAttributeAccepted(List<Attribute> attributes) {
```

Comment:
A high-level comment in front of this method is called for, to avoid someone needing to read and grok it to figure out its behavior.  This should be similar to whatever is put into the documentation as I mentioned above.